lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/interfaces.cpp

```diff
@@ -193,6 +192,16 @@ float VLH35_Angle_Bus::read_enc(uint8_t id) {
   return static_cast<float>(data) / static_cast<float>(1 << 16) * 360.0F;
 }
 
+Encoder AMT13_Angle_Bus::encs[NUM_ENCODERS] = {
+      Encoder(PIN_LIST[0], PIN_LIST[1]),
+      Encoder(PIN_LIST[2], PIN_LIST[3]),
+      Encoder(PIN_LIST[4], PIN_LIST[5])
+};
+
+float AMT13_Angle_Bus::read_enc(uint8_t id) {
+  return encs[id].read()/pulses_per_tick;
+}
+
```

Comment:
@WoozyDragon Note/todo: assuming this PR is just for adding driver code. will need to be careful as the API for old VLH encoders are angle in degrees rather than ticks, so it won't be a drop-in replacement. 

Given that the mapping from encoder position (ticks) to angular position (rad) should be constant, we should also make a constants.h, make a arduino script to calibrate by moving to a known  and populate the constants.h value. we can make this functionality a separate PR. thoughts?

Reply:
That variable is poorly named, it's the number of pulses in one full rotation of the encoder. Should be fairly easy to convert to degrees to make it a drop in replacement. 
Self-calibration might be nice, but we do have the datasheet for these encoders, so i can tell you the tics to rad conversion. (on current settings, 400/2pi ticks per rad, or roughly 63 ticks per rad)

Reply:
That being said, I just realized that these do not roll over (as in they just go up and up and up if you keep rotating in the same direction). That would be different from old encoders, although that may be beneficial? I'm not sure if we ever solved the spikey velocity graph around the rollover point, but this would solve that issue. 
We can talk more on Saturday